v7.h

```diff
@@ -23,7 +23,10 @@
 extern "C" {
 #endif /* __cplusplus */
 
+#include <stddef.h>   /* For size_t */
+
 #define V7_VERSION "1.0"
+#define V7_GENERIC_OBJECT_PROTOTYPE 0     /* For v7_create_object() */
```

Comment:
I'm sorry, I meant `(v7_create_undefined())`

Also `Object.create(null)` is a valid way to create a new object with a null prototype.